source/leuven/wice_advanced_guide.rst

```diff
@@ -145,7 +145,11 @@ Conda on wICE
 
 As the operating system and hardware are different on Genius and wICE, we advise
```

Comment:
As of monday 25/9/23, the OS of both machines will be identical. So, please update this line, too.